### PR TITLE
Add possibility to disable master password first time

### PR DESCRIPTION
**Pull request by fabio1999ita**
old vs new (the layout can still be improved)

image

### CODE REVIEW

**Comment by fabio1999ita** [comment]
video

**Comment by fabio1999ita** [comment]
video

**Comment by Pablete1234** [comment]
Currently you're using the Message in the jdialog for the string, then adding an "option" (usually, the list of buttons to show on bottom-right) that is a whole jpanel with everything

Instead, send just one jpanel as the message, which contains a jlabel, the field, and the button
lastly, make the button be the windows' default button, that way it will appear blue hinting the user that it should be their action to click

**Comment by fabio1999ita** [comment]
video

File: src/main/java/com/github/manolo8/darkbot/gui/login/SavedLogins.java [hunk 1]
```diff
@@ -109,14 +111,41 @@ private void updateList() {
...
+
+        check.addItemListener(e -> {
+            if (e.getStateChange() == ItemEvent.SELECTED) {
+                pass.setEditable(false);
+                button.setEnabled(true);
+            } else {
+                pass.setEditable(true);
+                if (new String(pass.getPassword()).isEmpty())
+                    button.setEnabled(false);
+            }
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/gui/login/SavedLogins.java:133]:
```suggestion
            pass.setEditable(e.getStateChange() != ItemEvent.SELECTED);
            button.setEnabled(pass.getPassword().length > 0 || e.getStateChange() == ItemEvent.SELECTED);
```

This logic can be cleaned up & simplified a bit

File: src/main/java/com/github/manolo8/darkbot/gui/login/SavedLogins.java [hunk 2]
```diff
@@ -109,14 +111,41 @@ private void updateList() {
...
+        });
+        pass.getDocument().addDocumentListener((GeneralDocumentListener) e -> button.setEnabled(!new String(pass.getPassword()).isEmpty() || check.isSelected()));
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/gui/login/SavedLogins.java:135]:
```suggestion
        pass.getDocument().addDocumentListener((GeneralDocumentListener) e -> 
                button.setEnabled(pass.getPassword().length > 0 || check.isSelected());
```
This line is too long to read comfortably, split it into a new line, and also avoid the new String allocation each time

**Review by Pablete1234 — CHANGES_REQUESTED** [review]
Just a couple final touchups but it looks good :+1: 

keep up the good work

**Pull request merged**